### PR TITLE
Added some Chinese metadata

### PR DESCRIPTION
**Pull request by Kotoki1337**
Special thanks:
@CKH0 Provides proofreading of traditional Chinese.
@uranami Assist in simplified Chinese Translation.
~~deadw1nter~~ Assist traditional Chinese Translation.

Chinese metadata translate Project Progress plz check [here](https://github.com/180TreehouseCSGO/Presences/blob/Kotoki1337-zh-translate-plan/README.md)

### CODE REVIEW

File: 巴哈姆特/dist/metadata.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "service": "巴哈姆特",
   "description": {
     "en": "The largest animation and game community website for Chinese, providing ACG daily news, popular rankings, and a wealth of discussion and communication space, as well as brilliant personal audio and video live and blog posts.",
-    "zh-TW": "華人最大動漫及遊戲社群網站，提供ACG 每日最新新聞、熱門排行榜，以及豐富的討論交流空間，還有精采的個人影音實況、部落格文章。"
+    "zh-TW": "華人最大動漫及遊戲社群網站，提供ACG 每日最新新聞、熱門排行榜，以及豐富的討論交流空間，還有精采的個人影音實況、部落格文章。",
+    "zh_CN": "华人最大动漫及游戏社群网站，提供ACG 每日最新新闻、热门排行榜，以及丰富的讨论交流空间，还有精彩的个人影音实况、部落格文章。"
```
Comment (Bas950) [inline 巴哈姆特/dist/metadata.json]: zh_TW not zh-TW

**Comment by Kotoki1337** [comment]
oh shit, i make a mistake

**Comment by Kotoki1337** [comment]
im sorry about mistake, but the problem has been solved

File: Rule34/dist/metadata.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
   "service": "Rule34",
   "description": {
     "en": "Rule 34 - If it exists, there's porn of it. If not, start uploading!",
-    "de": "Regel 34 - Wenn es existiert, gibt es Porn davon. Falls nicht, fang an etwas hochzuladen!"
+    "de": "Regel 34 - Wenn es existiert, gibt es Porn davon. Falls nicht, fang an etwas hochzuladen!",
+    "zh_CN": "Regel 34 - 这里在搞黄色，如果没有，那就上传!"
```
Comment (Bas950) [inline Rule34/dist/metadata.json]: This should probs be Rule or the chinese translation of Rule not the german version

**Comment by doomlerd** [comment]
Hello, are you planning to translate the rest presences ( the ones unchecked in the list ) ? because if that's the case we can wait until you've finished to merge it all at once, if not we'll proceed reviewing now.
Thank you for your contributions though, the whole staff team appreciates it!

**Comment by Kotoki1337** [comment]
> Hello, are you planning to translate the rest presences ( the ones unchecked in the list ) ? because if that's the case we can wait until you've finished to merge it all at once, if not we'll proceed reviewing now.
> Thank you for your contributions though, the whole staff team appreciates it!

Yes, we plan to complete all the translations, but due to the small number of our staff, the translation is very slow, so we have completed part of it for you to review. If there is no problem this time, we will refer to the translation as an example

**Pull request merged**